### ISSUE TITLE
Add flag to linter to show stderr into log messages

### ISSUE THREAD

**Issue by AbdealiJK** [coala-analyzer/coala#2711]
Sometimes the linter can fail due to reasons beyond our control. We should show the stderr from the linter in such cases as ERROR logs.
An example is https://github.com/coala-analyzer/coala-bears/issues/730 where the eslint config file could be wrong or even some files of eslint could be corrupt. Here, no error messages are shown by coala and it looks as if the linter parsed the file and no results are given (No results because stdout is empty)

We should add a `stderr_to_logs` or so argument

**Comment by Makman2** [comment]
I think a good idea is to check the return value of the executable. If it's not `0` then we could automatically print both, `stdout` and `stderr` as debug/error information :+1:
Then the user doesn't need to get familiar with another flag^^

**Comment by AbdealiJK** [comment]
Not sure of that's a good idea.
I remember that some of the linters will give warning messages and so on even though the exit code is not zero. For example of a config is being ignored for some reason.
Don't remember which tool and how to reproduce though

**Comment by Makman2** [comment]
alright then I would just print it always with a warning like:
```
'superlint' from XBear has written into stderr, though it was set as ignored:
ERROR: MEMORY EXPLOSION AT 0xFFFFFFFF
```
or else :3

**Comment by Makman2** [comment]
or we should really use another parameter that is by default true ^^

**Comment by sils1297** [comment]
@Makman2 any progress?

**Comment by Makman2** [comment]
not yet, is it really critical?

**Comment by sils** [comment]
Yep you hide errors and a lot of users are stumbling. This is important and should be fixed ASAP.
